charts/base-cluster/templates/ingress/nginx.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: ingress-nginx
+  namespace: ingress-nginx
+  labels: {{- include "common.labels.standard" $ | nindent 4 }}
+    app.kubernetes.io/component: ingress
+spec:
+  chart:
+    spec:
+      chart: ingress-nginx
+      sourceRef:
+        kind: HelmRepository
+        name: nginx
+        namespace: {{ .Release.Namespace }}
+      version: "4.x.x"
+  interval: 1m
+  {{- if .Values.monitoring.prometheus.enabled }}
+  dependsOn:
+    - name: kube-prometheus-stack-crds
+      namespace: monitoring
+  {{- end }}
+  values:
+    controller:
+      metrics:
+        enabled: {{ .Values.monitoring.prometheus.enabled }}
+        serviceMonitor:
+          enabled: {{ .Values.monitoring.prometheus.enabled }}
+          additionalLabels: {{- .Values.monitoring.labels | toYaml | nindent 12 }}
+    # TODO: toggle for PROXY protocol
+{{/*      config:*/}}
+{{/*        use-proxy-protocol: "true"*/}}
+    # TODO: allow fixed IP? better not
```

Comment:
Could you elaborate the fixed  IP?

Reply:
We _could_ allow the customer to set a custom, fixed, IP

But I personally rather have the customer be set up correctly with automation and not bind to a specific IP, what do you think?

Reply:
Yes, i think this would be the better solution.